tests/boot/boot_to_desktop.pm

```diff
@@ -35,6 +35,8 @@ sub run {
     # needs additional time too.
     $timeout += 60 if get_var('PATCH') || get_var('ONLINE_MIGRATION');
     $timeout += 60 if get_var('ENCRYPT');
+    # Add additional 60 seconds if the test suite is migration to newer Leap from older Leap
```

Comment:
Please make the comment explain *why* Leap should be treated differently

Reply:
it just takes longer than the default timeout... `2009 packages to upgrade, 195 to downgrade, 175 new, 20 to remove` looks normal to me. or you're asking me to look performance issue?

Reply:
Well, there is already `$timeout += 60 if get_var('PATCH') || get_var('ONLINE_MIGRATION');` Maybe just extend that to include `ZDUP`? I don't see why Leap should take longer than Tumbleweed or SLE to boot after a zdup upgrade so this is why I asked for the comment to explain better

Reply:
ok then. updated.